packages/yoroi-extension/app/api/ada/lib/storage/models/ConceptualWallet/index.js

```diff
@@ -79,6 +79,13 @@ export class ConceptualWallet implements IConceptualWallet, IRename {
   getWalletType: void => WalletType = () => {
     return this.walletType;
   }
+
+  getWalletVariant: void => 'web' | 'ledger' | 'trezor' = () => {
```

Comment:
Only nitpicking: "mnemonic" seems more accurate than "web"

Reply:
agreed, but that's just a refactoring of previously existing code so didn't want to break anything. Later can change. Especially after the merge of the DB PR